File: src/main/java/info/skyblond/archivedag/model/entity/ConfigEntity.java
package info.skyblond.archivedag.model.entity;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;

@Entity
@Table(name = "config_table")
@Getter
@Setter
@ToString
@RequiredArgsConstructor
public class ConfigEntity implements Serializable {
    @Id
    @Column(name = "key", updatable = false)
    String key;

    @Column(name = "value")
    String value;

    public ConfigEntity(String key, String value) {
        this.key = key;
        this.value = value;
    }
}

File: backend/src/main/kotlin/info/skyblond/archivedag/arstue/entity/ConfigEntity.kt
package info.skyblond.archivedag.arstue.entity

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "config_table")
class ConfigEntity(
    @Id
    @Column(name = "key", updatable = false)
    val key: String,

    @Column(name = "value")
    val value: String?
)
